OpenRA.Game/Network/OrderManager.cs

```diff
@@ -184,18 +192,39 @@ void CheckSync(byte[] packet)
 				syncForFrame.Add(frame, packet);
 		}
 
-		public bool IsReadyForNextFrame => GameStarted && pendingPackets.All(p => p.Value.Count > 0);
+		bool IsReadyForNextFrame => GameStarted && pendingPackets.All(p => p.Value.Count > 0);
+
+		int SuggestedTimestep
+		{
+			get
+			{
+				if (World == null)
+					return Ui.Timestep;
+
+				if (World.IsReplay)
+					return World.ReplayTimestep;
+
+				if (World.IsLoadingGameSave)
```

Comment:
This needs to stay above the `World.IsReplay` check, otherwise replays that started by loading a game save will take forever to start.